### PR TITLE
Adds Header Component

### PR DESCRIPTION
**Pull request by pantbibek21**
* Adds header component
* Removes existing unnecessary code structure
* Installs astro icons to use
* Updates the readme.md
* Adds global CSS styles

### CODE REVIEW

**Comment by manonja** [comment]
@pantbibek21 it's a small thing but since we don't have tests yet and that the main branch isn't protected, it would be awesome to include a screenshots showing the changes (mobile + desktop) in the pr description!

File: src/i18n/ui.ts [hunk 1]
```diff
@@ -1,25 +0,0 @@
-export const languages = {
```
Comment (manonja) [inline src/i18n/ui.ts:1]: don't you want to use these settings for the Nepalese translation?

File: src/components/Header.astro
Comment (manonja) [inline src/components/Header.astro:1]: looks great, awesome start!!

**Review by manonja — APPROVED** [review]

File: src/i18n/ui.ts (hunk 1 above)
Comment (pantbibek21) [reply src/i18n/ui.ts:1]: Yeah, I thought we could add that at the very end from the starter-kit!

**Comment by pantbibek21** [comment]
> @pantbibek21 it's a small thing but since we don't have tests yet and that the main branch isn't protected, it would be awesome to include a screenshots showing the changes (mobile + desktop) in the pr description! (only for UI changes)

@manonja yeah, that's a good suggestion, will do onwards!

**Comment by pantbibek21** [comment]
@manonja Really sorry, but this seems to be un-updated branch. Since there were multiple repo for the project, I mistakenly pushed from different repo. Closing this now and pushing the most updated one.

**Pull request closed without merging**